qa-tests-backend/src/test/groovy/IntegrationsTest.groovy

```diff
@@ -153,6 +153,8 @@ class IntegrationsTest extends BaseSpecification {
 
     @Unroll
     @Tag("BAT")
+    // "http://webhookserver.stackrox:8080": dial tcp: lookup webhookserver.stackrox on 172.30.0.10:53: no such host on P/Z
```

Comment:
remove these comments, since these are not relevant to tests executed via upstream